sound/soc/sof/pcm.c

```diff
@@ -215,6 +215,10 @@ static int sof_pcm_prepare(struct snd_pcm_substream *substream)
 	struct snd_sof_pcm *spcm;
 	int ret;
 
+	/* nothing todo for BE */
```

Comment:
@xiulipan i just fixed a bunch of these copy past errors. "todo" is not one word. it should be "to do". Otherwise the fix looks correct.

Reply:
@ranj063 
So should I fix this only or just fix all the `todo`? Since I copy paste it from the existing other function.